mesonbuild/interpreter/interpreter.py

```diff
@@ -2010,9 +2010,10 @@ def _validate_custom_target_outputs(has_multi_in: bool, outputs: T.Iterable[str]
         KwargInfo('feed', bool, default=False, since='0.59.0'),
         KwargInfo('capture', bool, default=False),
         KwargInfo('console', bool, default=False, since='0.48.0'),
+        KwargInfo('description', str, default='Generating {} with a custom command', since='1.4.0'),
```

Comment:
I am assuming that the placeholder gets filled in at some point later? I know you did work to enable this recently.

Reply:
Yes. This is the name of the output file. I added it to the documentation.

Reply:
I wonder if it would instead be better to use `@FILENAME@` syntax instead. Here we are exposing Python internals, which isn't great. This also allows us to be extensible in the future if we want to expose other placeholders.

Reply:
👍 That's a very good point!

Reply:
I don't know if FILENAME is a great identifier. Maybe something like `@OUTPUT@` which is relative to the build dir?

Reply:
Just updated, using `@TARGET@` placeholder.